pkg/apis/planetscale/v2/vitesscluster_defaults.go

```diff
@@ -22,13 +22,27 @@ import (
 	"k8s.io/utils/pointer"
 )
 
+var (
+	defaultTopologyReconciliationConfig = TopoReconcileConfig{
```

Comment:
So far, the pattern has been to keep the concrete default values in `defaults.go`, and the `*_defaults.go` files just have methods to fill in those concrete values. We might actually rethink the single `defaults.go` file just like we broke up `helpers.go` but we should keep things consistent until we decide to refactor.